PMA-frontend/mainGUI.py

```diff
@@ -3,40 +3,235 @@
+        self.im_priceEntered.grid(column=2, row=6, padx=5, pady=5)
+        self.im_quantityEntered             = Entry(self.im_insert, width=20, textvariable=self.im_quantity)
+        self.im_quantityEntered.grid(column=2, row=7, padx=5, pady=5)
+
+        self.im_updateButton1 = Button(self.im_insert, text='Confirm',
+                                            command=lambda: self.medInsertConfirmedClicked(self.insertValues))
+        self.im_updateButton1.grid(column=2, row=8, padx=5, pady=5)
+
+        ##Patient
+        self.insert_patient_label1 = Label(self.ip_insert, text="Enter The Patient ID*")
+        self.insert_patient_label1.grid(column=1, row=1)
+        self.insert_patient_label2 = Label(self.ip_insert, text="Enter The Patient Name")
+        self.insert_patient_label2.grid(column=1, row=2)
+        self.insert_patient_label3 = Label(self.ip_insert, text="Enter The Patient Allergy")
+        self.insert_patient_label3.grid(column=1, row=3)
+
+        self.ip_patient_ID = StringVar()
+        self.ip_patient_name_input = StringVar()
+        self.ip_patient_allergy_input = StringVar()
+
+        self.ip_input_list1 = [self.ip_patient_ID, self.ip_patient_name_input, self.ip_patient_allergy_input
+                               ]
+
+        self.ip_patient_IDEntered = Entry(self.ip_insert, width=20, textvariable=self.ip_patient_ID)
+        self.ip_patient_IDEntered.grid(column=2, row=1, padx=5, pady=5)
+        self.ip_patient_name_inputEntered = Entry(self.ip_insert, width=20, textvariable=self.ip_patient_name_input)
+        self.ip_patient_name_inputEntered.grid(column=2, row=2, padx=5, pady=5)
+        self.ip_patient_allergy_inputEntered = Entry(self.ip_insert, width=20, textvariable=self.ip_patient_allergy_input)
+        self.ip_patient_allergy_inputEntered.grid(column=2, row=3, padx=5, pady=5)
+
+        self.ip_updateButton1 = Button(self.ip_insert, text='Confirm',
+                                       command=lambda: self.patientInsertConfirmedClicked(self.insertValues))
+        self.ip_updateButton1.grid(column=2, row=8, padx=5, pady=5)
+        ##RECEIPT
+        self.insert_r_label1 = Label(self.ir_modifyframe, text="Enter Your Receipt ID")
+        self.insert_r_label1.grid(column=1, row=1, padx=5, pady=5)
+        self.insert_r_label2 = Label(self.ir_modifyframe, text="Enter Your Total Purchase")
+        self.insert_r_label2.grid(column=1, row=2, padx=5, pady=5)
+        self.insert_r_label3 = Label(self.ir_modifyframe, text="Enter The Patient ID")
+        self.insert_r_label3.grid(column=1, row=3, padx=5, pady=5)
+        self.insert_r_label4 = Label(self.ir_modifyframe, text="Enter The Receipt Date (YYYY-MM-DD)")
+        self.insert_r_label4.grid(column=1, row=4, padx=5, pady=5)
+        self.insert_r_label5 = Label(self.ir_modifyframe, text="How Many Drugs Info Needed To Be Inserted? (Max: 5)")
+        self.insert_r_label5.grid(column=1, row=5, padx=5, pady=5)
+
+        self.rid_input = StringVar()
```

Comment:
not appropriate variable name, use self.insert_rid_input...

Reply:
yeah my bad. Missed that one